exchange/exchangetest/passthrough.json

```diff
@@ -0,0 +1,122 @@
+{
+    "passthrough_ext_flag": true,
+    "incomingRequest": {
+        "ortbRequest": {
+            "id": "some-request-id",
+            "site": {
+                "page": "test.somepage.com"
+            },
+            "imp": [{
+                "id": "my-imp-id",
+                "video": {
+                    "mimes": ["video/mp4"]
+                },
+                "ext": {
+                    "appnexus": {
+                        "placementId": 1
+                    },
+                    "prebid": {
+                        "passthrough": {
+                            "imp_passthrough_val": 20
+                        }
+                    }
+                }
+            }],
+            "ext": {
+                "prebid": {
+                    "passthrough": {"bid_response_passthrough":20}
+                }
+            }
+        }
+    },
+    "outgoingRequests": {
+        "appnexus": {
+            "expectRequest": {
+                "ortbRequest": {
+                    "id": "some-request-id",
+                    "site": {
+                        "page": "test.somepage.com"
+                    },
+                    "imp": [{
+                        "id": "my-imp-id",
+                        "video": {
+                            "mimes": ["video/mp4"]
+                        },
+                        "ext": {
+                            "bidder": {
+                                "placementId": 1
+                            },
+                            "prebid": {
+                                "passthrough": {
+                                    "imp_passthrough_val": 20
+                                }
+                            }
+                        }
+                    }],
+                    "ext": {
+                        "prebid": {
+                            "passthrough": {"bid_response_passthrough":20}
+                        }
+                    }
+                }
+            },
+            "mockResponse": {
+                "pbsSeatBids": [{
+                    "pbsBids": [{
+                        "ortbBid": {
+                            "id": "apn-bid",
+                            "impid": "my-imp-id",
+                            "price": 0.3,
+                            "w": 200,
+                            "h": 250,
+                            "crid": "creative-1",
+                            "ext": {
+                                "someField": "someValue",
+                                "origbidcpm": 0.3,
+                                "prebid": {
+                                    "willBeOverwritten": "by core logic",
+                                    "passthrough": {
```

Comment:
The `mockResponse` is the mocked response from the bidder. This might not have the `passthrough` object in the bid back. This only needs to be asserted in the response from Prebid

Reply:
Yeah you're right, I'll remove this piece